### ISSUE TITLE
provide a link to the website under "about" in the readme

### ISSUE THREAD

**Issue by wiki-me** [Tuxemon/Tuxemon#1227]
**Is your feature request related to a problem? Please describe.**
It is a little too easy to miss the tuxemon website on the read me.

**Describe the solution you'd like**
provide a link to the website under "about", see the read me of [thrive](https://github.com/Revolutionary-Games/Thrive) for an example.

**Describe alternatives you've considered**
none.

**Comment by AndyMender** [comment]
Which website do you mean specifically? https://www.tuxemon.org? 

The README currently has an External Links section with the link to the website and other resources.

@kerizane do you think it would make sense to move these next to the Build Status and Documentation Status badges?

**Comment by wiki-me** [comment]
Yeah that's the website (is there something else?)

I think it's better to put it under about (It probably fits better github users visual pattern searching).

**Comment by kerizane** [comment]
Good idea, but the website is quite out of date, so I don't want to promote it atm.

**Comment by wiki-me** [comment]
I went over the website and didn't see anything out of date, but anyway the website is already linked to at the end of the read me, as well as in external websites (slant.co, openhub and alternativeto.net).
